agent/consul/state/catalog_test.go

```diff
@@ -2107,7 +2107,7 @@ func TestStateStore_ConnectServiceNodes(t *testing.T) {
 	ws = memdb.NewWatchSet()
 	idx, nodes, err = s.ConnectServiceNodes(ws, "db", nil)
 	assert.Nil(err)
-	assert.Equal(idx, uint64(idx))
```

Comment:
Something tells me that the original author didn't mean to compare `idx` with itself.

Reply:
Good catch! I missed this. I'll have another look at what was intended here. I guess the value is predictable, so I may be able to set it to a literal int value.